app/services/get-overview.js

```diff
@@ -17,33 +17,37 @@ module.exports = function (id, organisationLevel) {
 
   result.breadcrumbs = getBreadcrumbs(id, organisationLevel)
   return overviewPromise.then(function (results) {
-    result.overviewDetails = calculateValues(results)
+    result.overviewDetails = calculateValues(results, isCSV)
     result.title = result.breadcrumbs[0].title
     result.subTitle = organisationalUnitType.displayText
     return result
   })
 }
 
-var calculateValues = function (results) {
-  var totals = { name: 'Total / Average', totalPercentage: 0, totalPoints: 0, totalContractedHours: 0, totalReduction: 0, totalTotalCases: 0 }
+var calculateValues = function (results, isCSV) {
+  var totals = { name: 'Total / Average', totalPercentage: 0, totalPoints: 0, totalContractedHours: 0, totalReduction: 0, totalTotalCases: 0, totalRemainingPoints: 0 }
   var totalsToReturn = {}
   if (results.length !== undefined) {
     results.forEach(function (result) {
       result.capacityPercentage = 0
       if (result.availablePoints > 0) {
         result.capacityPercentage = (result.totalPoints / result.availablePoints) * 100
+        result.remainingPoints = result.availablePoints - result.totalPoints
```

Comment:
Think this can go outside the if. If they have zero available points but cases assigned, that should probably be displayed as a negative remaining points

Reply:
Okay, cool. I wasn't too sure. Will do that now.